test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
```

Comment:
you added the rspec but the old test dir still in the project ^^ run this command `rm -rf test` to remove it :)